spec/lib/video_info_spec.rb

```diff
@@ -70,6 +70,13 @@
     end
   end
 
+  describe ".provider_api_keys" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.